### PR TITLE
Adds feature to check tags

### LINKED ISSUE #44: Tags as a non-array causes issues
Not using an array for tags causes a 500 error on the forge.

Opened a ticket to make this behaviour nicer on the Forge side: https://tickets.puppetlabs.com/browse/FORGE-349

In the mean-time, would be good to add as a feature to the linter 😄

### PR DESCRIPTION
**Pull request by petems**
* Tags have to be in an array format, adds simple check for this

Closes #44

### CODE REVIEW

**Comment by domcleal** [comment]
Could you fix the two rubocop failures please?

Regarding the module line length, I think you could safely disable that for now until the module's refactored.

**Comment by petems** [comment]
@domcleal Done 👍

File: tests/tags_no_array/metadata.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+  "name": "puppetlabs-postgresql",
+  "version": "3.4.1",
+  "author": "Inkling/Puppet Labs",
+  "summary": "PostgreSQL defined resource types",
+  "license": "Apache-2.0",
+  "source": "git://github.com/puppetlabs/puppet-postgresql.git",
+  "project_page": "https://github.com/puppetlabs/puppet-postgresql",
+  "issues_url": "https://github.com/puppetlabs/puppet-postgresql/issues",
+  "tags": "postgres",
```
Comment (petems) [inline tests/tags_no_array/metadata.json]: Seems sensible, will do!

Comment (petems) [inline tests/tags_no_array/metadata.json]: Happy path test added! 👍

File: tests/test.sh [hunk 2]
```diff
@@ -89,6 +89,12 @@ test "proprietary" $SUCCESS
 # Run without a metadata.json or Rakefile, expect FAILURE
 test "no_files" $FAILURE
 
+# Run with tags not in an array in metadata.json, expect FAILURE
+test "tags_no_array" $FAILURE
+
+# Run with tags in an array in metadata.json, expect SUCCESS
+test "tags_no_array" $SUCCESS
```
Comment (domcleal) [inline tests/test.sh]: `tags_with_array`

Comment (juniorsysadmin) [inline tests/test.sh]: typo

**Comment by petems** [comment]
Thanks everyone 👍

**Pull request merged**